app/views/TaxYearOverview.scala.html

```diff
@@ -73,48 +73,55 @@
                         }
-                    </div>
-                        @if(documentDetailWithDueDate.dunningLock){<div class="form-hint govuk-body-s"> @messages("tax-year-overview.payments.paymentUnderReview") </div>}
-                        @if(documentDetailWithDueDate.documentDetail.hasLpiWithDunningBlock){<div class="form-hint govuk-body-s"> @messages("tax-year-overview.payments.paymentUnderReview") </div>}
-                    </td>
-                    @if(documentDetailWithDueDate.isLatePaymentInterest && documentDetailWithDueDate.documentDetail.latePaymentInterestAmount.isDefined) {
-                        <td class="govuk-table__cell">@documentDetailWithDueDate.dueDate.getOrElse(throw new MissingFieldException("Due Date")).toLongDateShort</td>
-                        <td class="govuk-table__cell">@messages(s"tax-year-overview.payments.${documentDetailWithDueDate.documentDetail.getInterestPaidStatus}")</td>
-                        <td class="govuk-table__cell">@documentDetailWithDueDate.documentDetail.latePaymentInterestAmount.getOrElse(throw new MissingFieldException("Late Payment Interest Amount")).toCurrencyString</td>
-                    } else if(codingOutEnabled && documentDetailWithDueDate.documentDetail.isPayeSelfAssessment) {
+                    }
+                } else if(codingOutEnabled && documentDetailWithDueDate.documentDetail.isPayeSelfAssessment) {
                         <td class="govuk-table__cell">@messages("tax-year-overview.na")</td>
                         <td class="govuk-table__cell">@messages("tax-year-overview.paye-tax-code")</td>
-                        <td class="govuk-table__cell">@documentDetailWithDueDate.documentDetail.amountCodedOut.getOrElse(throw new MissingFieldException("Amount Coded Out")).toCurrencyString</td>
-                    } else {
-                        <td class="govuk-table__cell">@documentDetailWithDueDate.dueDate.getOrElse(throw new MissingFieldException("Due Date")).toLongDateShort</td>
+                        <td class="govuk-table__cell">@documentDetailWithDueDate.documentDetail.amountCodedOut.getOrElse(throw MissingFieldException("Amount Coded Out")).toCurrencyString</td>
+                } else {
+                        <td class="govuk-table__cell">@documentDetailWithDueDate.dueDate.getOrElse(throw MissingFieldException("Due Date")).toLongDateShort</td>
                         <td class="govuk-table__cell">@messages(s"tax-year-overview.payments.${documentDetailWithDueDate.documentDetail.getChargePaidStatus}")</td>
-                        <td class="govuk-table__cell">@documentDetailWithDueDate.documentDetail.originalAmount.getOrElse(throw new MissingFieldException("Original Amount")).toCurrencyString</td>
-                    }
-                </tr>
-            }
-            </tbody>
-        </table>
-    }
+                        @documentDetailWithDueDate.documentDetail.originalAmount.map { originalAmount =>
```

Comment:
Not sure why its changed from last review but this one needs throwing exception as well.

Reply:
All should be done now. Thanks